### PR TITLE
[full-ci][tests-only] use oCIS from the cache used from multiple repos

### PR DESCRIPTION
**Pull request by kiranparajuli589**
**Description**
With this PR, the ocis binary is cached inside the `cache_public_s3_bucket` as
```
s3/$CACHE_BUCKET/ocis-build/$OCIS_COMMITID
|__ ocis (binary)
``` 
The same setup is to be made on the owncloud/owncloud-sdk CI, so that, if both repositories. uses the same commit id for oCIS, the binary can be shared.

**Related Issue:**
- Part of https://github.com/owncloud/QA/issues/756
- With https://github.com/owncloud/owncloud-sdk/pull/1137

---
Signed-off-by: Kiran Parajuli <kiranparajuli589@gmail.com>

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oC10</strong> https://drone.owncloud.com/owncloud/web/27817/11/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/27817/tracing/file-alice-2022-8-25-09-09-08.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/27817/tracing/file-brian-2022-8-25-09-09-16.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/27817/tracing/folder-alice-2022-8-25-09-07-38.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/27817/tracing/folder-brian-2022-8-25-09-07-45.zip 


</p></details>

File: .drone.star [hunk 1]
```diff
@@ -2405,112 +2454,72 @@ def cacheOcisPipeline(ctx):
...
-            },
-            "AWS_SECRET_ACCESS_KEY": {
-                "from_secret": "cache_s3_secret_key",
-            },
-        },
+        "environment": minio_mc_environment,
         "commands": [
-            "mc alias set s3 $MC_HOST $AWS_ACCESS_KEY_ID $AWS_SECRET_ACCESS_KEY && mc find s3/owncloud/web/ocis-build",
+            ". ./.drone.env",
+            "pwd && ls -la %s/$OCIS_COMMITID" % dir["base"],
```
Comment (grgprarup) [inline .drone.star:2519]:
```suggestion
```
Do we need this?

Comment (kiranparajuli589) [reply .drone.star]: nope. removed

**Review by grgprarup — COMMENTED** [review]
lgtm

**Review by individual-it — APPROVED** [review]

**Review by individual-it — CHANGES_REQUESTED** [review]
why moving the files one level higher up?

**Comment by kiranparajuli589** [comment]
> why moving the files one level higher up?

yes previously it was under the `web` directory, but now the oCIS cache is saved irrespective of repositories caching them. so getting it out from the `web` allows us to use the same cache from SDK or any other repo CI. The same location is used in the owncloud SDK CI so that they can use the same resource if they're trying to get oCIS of same commitID

**Review by individual-it — APPROVED** [review]

**Pull request merged**